mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/CassandraMessageManager.java

```diff
@@ -44,9 +45,11 @@
 
     public CassandraMessageManager(MailboxSessionMapperFactory mapperFactory, MessageSearchIndex index, 
             MailboxEventDispatcher dispatcher, MailboxPathLocker locker, Mailbox mailbox, QuotaManager quotaManager, 
-            QuotaRootResolver quotaRootResolver, MessageParser messageParser, MessageId.Factory messageIdFactory) throws MailboxException {
+            QuotaRootResolver quotaRootResolver, MessageParser messageParser, MessageId.Factory messageIdFactory,
+            FetchBatchSizes fetchBatchSizes) throws MailboxException {
         super(mapperFactory, index, dispatcher, locker, mailbox, new UnionMailboxACLResolver(), 
                 new SimpleGroupMembershipResolver(), quotaManager, quotaRootResolver, messageParser, messageIdFactory);
+        setFetchBatchSizes(fetchBatchSizes);
```

Comment:
Much more complexe, we then should handle the loading of those parameters by adding it to mailbox.xml (for example).
and then change the spring xml files...